package/com.unity.formats.usd/Dependencies/USD.NET.Unity/Shading/UsdPreviewSurface/PrimvarReaderSample.cs

```diff
@@ -85,4 +79,38 @@ public class Outputs : SampleBase
         [UsdNamespace("outputs")]
         public Outputs outputs = new Outputs();
     }
+
+    // In 21.11+ the PrimvarReader varname was switched to a string from a TfToken. As a TfToken can be cast to a string, in order to support older USD versions as
+    // well as newer, we treat the varname as a string. However, so that exported shaders fit the USD spec for 20.08 (the version included in this package) we have
+    // a different version of the class for export that uses a TfToken (see below). This should be removed when we upgrade our USD version.
+    [System.Serializable]
+    [UsdSchema("Shader")]
+    public class PrimvarReaderImportSample<T> : PrimvarReaderSample<T> where T : struct
+    {
+        /// <summary>
+        /// Name of the primvar to be read from the primitive as a string.
+        /// </summary>
+        /// <remarks>
+        /// In USD 21.11 the varname was switched from a TfToken to a string. However, as a TfToken can be implicitly converted to a string,
+        /// making this a string type works for both during *import*.
+        /// </remarks>
+        [InputParameter("_Varname")]
+        public Connectable<string> varname = new Connectable<string>("st");
```

Comment:
I take it st is a default value here? I'm not sure it makes much sense for it to be a default in this context?